browser/ui/views/toolbar/brave_app_menu.cc

```diff
@@ -41,6 +55,17 @@ void BraveAppMenu::RunMenu(views::MenuButtonController* host) {
 }
 
 void BraveAppMenu::ExecuteCommand(int command_id, int mouse_event_flags) {
+  // Suspect that the entry is null but can't imagine which command causes it.
+  // See
+  // https://github.com/brave/brave-browser/issues/37862#issuecomment-2078553575
+  if (!IsBookmarkCommand(command_id) && command_id != IDC_EDIT_MENU &&
+      command_id != IDC_ZOOM_MENU &&
+      command_id_to_entry_.find(command_id) == command_id_to_entry_.end()) {
+    LOG(ERROR) << __func__ << " entry should be exsit for " << command_id;
```

Comment:
@simonhong you can add a crash key with command_id or other parameters, see https://source.chromium.org/chromium/chromium/src/+/main:base/debug/crash_logging.h;l=26?q=CrashKeyString&ss=chromium%2Fchromium%2Fsrc

Reply:
Nice. Added.

Reply:
this was `#define IDC_SHOW_BOOKMARK_SIDE_PANEL    40274`
see https://share.backtrace.io/api/share/d8b80FNXbVwx2J4JkpClhV1

Reply:
`IDC_SHOW_BOOKMARK_SIDE_PANEL` is added from `BookmarkSubMenuModel` and it should be added to `command_id_to_entry_` at AppMenu::AddMenuItem().
Still don't know how `command_id_entry_` doesn't have it.